Documentation/adding-new-repository.md

```diff
@@ -41,6 +41,30 @@ When applicable, prepare Docker image repositories for the new component.
   * robots: Read & Write
 * For closed source repository, create a new private repository in registry.mender.io
 
+## Makefile
+
+Consider adding a Makefile to the repository. This will serve both developers on their
+local environment and automated templated jobs on CI.
+
+A Makefile in a Mender repository is expected to contain at least the following targets:
+
+* `fmt`: Format the code. Typically using `gofmt` for golang repositories
+* `lint`: Lint the code. On new golang repositories, we are trying to use `golangci-lint` tool.
+* `build`: Build repository binary in `bin/` directory.
+* `test`: Run unit tests and generate code coverage metrics in file `coverage.txt`.
+
+Other useful targets are:
+
+* `docs`: Auto-generate Python test classes for tests
+* `docker`: Build Docker production image
+* `build-test`: Build binary with code coverage instrumentation
```

Comment:
Question:
Should all these be prefixed by `test` instead?

Reply:
I am trying to document [what we already have](https://github.com/mendersoftware/deviceconnect/blob/master/Makefile), as we decided in last QA Arch meeting. I think renaming is too much work now...

Reply:
Oh, I see, far out of scope!